applications/app/views/survey/thankyou.scala.html

```diff
@@ -0,0 +1,21 @@
+@(surveyPage: Page)(implicit request: RequestHeader)
```

Comment:
All the survey pages have `<meta name="robots" content="noindex">` because of `shouldGoogleIndex` on the `Survey` page being set to `false`.